core/src/main/java/org/opensearch/sql/executor/execution/PaginatedQueryService.java

```diff
@@ -46,11 +46,7 @@ public void executePlan(LogicalPlan plan,
    */
   public void executePlan(PhysicalPlan plan,
                           ResponseListener<ExecutionEngine.QueryResponse> listener) {
-    try {
-      executionEngine.execute(plan, ExecutionContext.emptyExecutionContext(), listener);
-    } catch (Exception e) {
-      listener.onFailure(e);
-    }
+    executionEngine.execute(plan, ExecutionContext.emptyExecutionContext(), listener);
```

Comment:
Why was the try-catch block removed? Was something changed so that we don't expect this to throw an exception anymore?

Reply:
`executePlan` called from `execute` function only which already has try-catch block.